apps/meteor/client/NavBarV2/NavBarPagesGroup/actions/CreateChannelModal.tsx

```diff
@@ -339,13 +344,16 @@ const CreateChannelModal = ({ teamId = '', onClose, reload }: CreateChannelModal
 							</Field>
 							<Field>
 								<FieldRow>
-									<FieldLabel htmlFor={broadcastId}>{t('Broadcast')}</FieldLabel>
```

Comment:
why this change is necessary? Why was necessary to add IDs on those components? Can you explain that better to me?

Reply:
While writing tests, getByLabelText was failing because aria-labelledby pointed to an invalid ID. Instead of just removing the ARIA attributes, I added explicit IDs so the association became valid. This fixed the test errors and also improved accessibility, since screen readers can now properly link labels to their controls.